sites/fast-website/src/public/index.html

```diff
@@ -1,129 +1,282 @@
+                            <fast-badge
+                                fill="primary"
+                                color="primary"
+                                slot="badge"
+                                class="section-badge"
+                                >LEARN, MODIFY, AND FOLLOW</fast-badge
+                            >
+                            <h2 class="section-heading">Join the community</h2>
+                            <p slot="body" class="section-paragraph">
+                                Our community drives the rhythm and direction of FAST. By
+                                taking part, your voice will help shape our future.
+                            </p>
+                        </site-section-header>
+                        <fast-divider
+                            class="section-decoration"
+                            role="presentation"
+                        ></fast-divider>
+                        <site-content-placement-container
+                            section="community"
+                        ></site-content-placement-container>
+                    </div>
+                </section>
 
-        <aside class="side-nav-container">
-          <site-side-navigation category="links" class="social-links"></site-side-navigation>
-          <site-side-navigation category="scroll" class="section-scroll"></site-side-navigation>
-        </aside>
+                <aside class="side-nav-container">
+                    <site-side-navigation
+                        category="links"
+                        class="social-links"
+                    ></site-side-navigation>
+                    <site-side-navigation
+                        category="scroll"
+                        class="section-scroll"
+                    ></site-side-navigation>
+                </aside>
 
-        <fast-divider class="footer-divider" role="seperator"></fast-divider>
-        <footer class="footer">
-          <site-navigation>
-            <ul>
-              <li><fast-anchor appearance="lightweight" href="https://github.com/microsoft/fast/blob/master/LICENSE">License</fast-anchor></li>
-              <li><fast-anchor appearance="lightweight" href="https://privacy.microsoft.com/en-US/privacystatement">Privacy & Cookies</fast-anchor></li>
-              <li><fast-anchor appearance="lightweight" href="https://www.microsoft.com/en-us/legal/intellectualproperty/copyright/default.aspx">Terms of Use</fast-anchor></li>
-              <li><p>&copy; 2020 Microsoft</p></li>
-            </ul>
-          </site-navigation>
-          <fast-anchor class="logo" appearance="lightweight" href="/" aria-label="Home">
-            <%= require("svg/icon-brand.svg") %>
-          </fast-anchor>
-        </footer>
-      </div>
-      <background-design></background-design>
-    </fast-design-system-provider>
-  </body>
+                <fast-divider class="footer-divider" role="separator"></fast-divider>
```

Comment:
I did the change here, prettier is responsible for all the other changes.

Reply:
Thanks! I didn't think we had prettier running on our commit hook. Out of curiosity, did this run on the hook or did you run prettier manually?

Reply:
It was running when I saved the file, I can go ahead and turn off formant on save option of it is not running in pre commit hook

Reply:
I wouldn't worry about it! I think we'll set it up to run on HTML either way. :)

Only thing I noticed is that in this implementation we could probably just drop the role as `separator` is implicit on Divider.

Reply:
I removed the role and verified it, it works fine, thanks for me letting know 👍

Reply:
Thanks @abhishekjakhar for your contribution.